docs/sql-ref-ansi-compliance.md

```diff
@@ -110,19 +110,20 @@ Besides, the ANSI SQL mode disallows the following type conversions which are al
-| Struct        | N                                    | Y                                    | N                                    | N                                    | N                                    | N                                    | N                                    | N      | N                                    | N                                    | <span style="color:red">**Y**</span> |
+| Source\Target | Numeric                              | String                               | Date                                 | Time | Timestamp                            | Timestamp_NTZ                        | Interval                             | Boolean                              | Binary | Array                                | Map                                  | Struct                               |
+|---------------|--------------------------------------|--------------------------------------|--------------------------------------|------|--------------------------------------|--------------------------------------|--------------------------------------|--------------------------------------|--------|--------------------------------------|--------------------------------------|--------------------------------------|
+| Numeric       | <span style="color:red">**Y**</span> | <span style="color:red">**Y**</span> | N                                    | N    |<span style="color:red">**Y**</span> | N                                    | <span style="color:red">**Y**</span> | Y                                    | N      | N                                    | N                                    | N                                    |
+| String        | <span style="color:red">**Y**</span> | Y                                    | <span style="color:red">**Y**</span> | Y    |<span style="color:red">**Y**</span> | <span style="color:red">**Y**</span> | <span style="color:red">**Y**</span> | <span style="color:red">**Y**</span> | Y      | N                                    | N                                    | N                                    |
+| Date          | N                                    | Y                                    | Y                                    | N    | Y                                   | Y                                    | N                                    | N                                    | N      | N                                    | N                                    | N                                    |
+| Time          | N                                    | Y                                    | N                                    | N    | N                                   | N                                    | N                                    | N                                    | N      | N                                    | N                                    | N                                    |
```

Comment:
Casting from Time to Time should be allowed, right?

Reply:
I do believe the table should reflect actual information, but not potential support. At the moment, the casting from TIME(n) to TIME(m) (m could be different from n) has not been supported yet. There is the task for that: https://issues.apache.org/jira/browse/SPARK-52618

Reply:
> But casting from TIME(n) to TIME(n) is supported...

Yep.

> Shall we keep the doc open until https://issues.apache.org/jira/browse/SPARK-52618 is done?

ok, let's keep it open till the task is done.